src/avatar/stories/Avatar.stories.tsx

```diff
@@ -101,30 +119,32 @@ export const Group = storyTemplate<AvatarProps>(
   { size: "xl" },
 )({});
 
-export const GroupWithLimit = () => {
-  const sizes = ["xs", "sm", "md", "lg", "xl"] as const;
-  return (
-    <>
-      {sizes.map((size, i) => {
-        return (
-          <AvatarGroup
-            key={i}
-            limit={i + 1}
-            className="flex mt-4"
-            size={size}
-            showBorder
-            borderColor="border-red-200"
-          >
-            <Avatar name="Dan Abrahmov" src="https://bit.ly/dan-abramov" />
-            <Avatar name="Anurag Hazra" />
-            <Avatar name="Kent Dodds" src="https://bit.ly/kent-c-dodds" />
-            <Avatar />
-            <Avatar name="Navin Moorthy" />
-            <Avatar name="Ryan Florence" src="https://bit.ly/ryan-florence" />
-            <Avatar name="Christian Nwamba" src="https://bit.ly/code-beast" />
-          </AvatarGroup>
-        );
-      })}
-    </>
-  );
-};
+export const GroupWithLimit = storyTemplate<AvatarGroupProps>(
+  ({ size, ...args }) => {
+    const sizes = ["xs", "sm", "md", "lg", "xl"] as const;
+    return (
+      <>
+        {sizes.map((size, i) => {
```

Comment:
You can drop the size like you said on the Friday.

Since we have the args now !

Reply:
No, i said sizes are fine since they are already nicely laid out in smaller to bigger. i deliberately left the sizes untouched.